partiql-planner/src/main/kotlin/org/partiql/planner/internal/ProblemGenerator.kt

```diff
@@ -40,16 +40,16 @@ internal object ProblemGenerator {
     }
 
     fun missingRex(causes: List<Rex.Op>, problem: Problem): Rex =
-        rex(StaticType.MISSING, rexOpMissing(problem, causes))
+        rex(StaticType.ANY, rexOpMissing(problem, causes))
 
     fun missingRex(causes: Rex.Op, problem: Problem): Rex =
-        rex(StaticType.MISSING, rexOpMissing(problem, listOf(causes)))
+        rex(StaticType.ANY, rexOpMissing(problem, listOf(causes)))
 
     fun errorRex(causes: List<Rex.Op>, problem: Problem): Rex =
-        rex(StaticType.MISSING, rexOpErr(problem, causes))
+        rex(StaticType.ANY, rexOpErr(problem, causes))
 
     fun errorRex(trace: Rex.Op, problem: Problem): Rex =
-        rex(StaticType.MISSING, rexOpErr(problem, listOf(trace)))
+        rex(StaticType.ANY, rexOpErr(problem, listOf(trace)))
```

Comment:
Hmm. Good question. I'm unsure. `unknown` is used for literal values whose types need to be inferred based on their context (and original value).

Just thinking out loud here, but the goal of the planner is to produce a plan that can be "almost" directly executed. Given our gradual nature, the execution must execute the same way as a fully typed plan would have. Therefore, if `unknown` is treated in such a way that it may be handled in function resolution, then I think it would be appropriate.

Maybe that is appropriate in some scenarios. But I'm also thinking about functions that always receive the missing value as one of its inputs. I imagine that, currently, this probably collapses as a problem to always return missing -- though, it should be typed as the original function's type.